src/Sylius/Bundle/GridBundle/Doctrine/PHPCRODM/ExpressionBuilder.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\GridBundle\Doctrine\PHPCRODM;
+
+use Sylius\Component\Grid\Data\ExpressionBuilderInterface;
+use Doctrine\ODM\PHPCR\Query\Builder\QueryBuilder;
+use Doctrine\Common\Collections\ExpressionBuilder as CollectionsExpressionBuilder;
+
+/**
+ * ExpressionBuilder which builds Doctrine Commons
+ * ExpressionBuilder object graph which we can then walk in order
+ * to build up the PHPCR-ODM query builder.
+ */
+class ExpressionBuilder implements ExpressionBuilderInterface
+{
+    private $expressionBuilder;
+    private $orderBys = [];
+
+    function __construct()
+    {
+        $this->expressionBuilder = new CollectionsExpressionBuilder();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function andX($expressions)
+    {
+        return $this->expressionBuilder->andX($expressions);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function orX($expressions)
+    {
+        return $this->expressionBuilder->orX($expressions);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function comparison($field, $operator, $value)
+    {
+        throw new \BadMethodCallException('Not supported yet.');
```

Comment:
Nowhere yet, but I wanted to have it on the interface. Let's keep it for now like for other drivers.